docs: update documentation related to dependabot (#740)
ansys/pyansys-dev-guide: doc/source/how-to/packaging.rst

Dependabot allows for two different types of updates:

* **Dependabot version updates**: Automated pull requests that keep dependencies updated,
  even when they don't have any vulnerabilities. To check the status of version updates,
  navigate to the **Insights** tab of your repository and then select **Dependency Graph**
  and **Dependabot**.
* **Dependabot security updates**: Automated pull requests that help update

>   dependencies with known vulnerabilities.

Reviewer: Maybe we could reference the DB that is used to list known vulnerabilities so people knows where it comes from.

`Enabling or disabling Dependabot security updates for an individual repository
<https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#enabling-or-disabling-dependabot-security-updates-for-an-individual-repository>`_
in the GitHub documentation.

.. caution::

    In the past, dependabot security updates were enabled by default for public repositories.

>     However, this has evoled over time and you might need to enable it manually for your repository.

Reviewer 2: ```suggestion
    However, this has evolved over time and you might need to enable it manually for your repository.
```

>     However, this has evolved over time and you might need to enable it manually for your repository.

Reviewer 3: ```suggestion
    However, this behavior has changed over time, and you may now need to enable it manually for your repository.
```
